src/odemis/util/test/transform_test.py

```diff
@@ -649,5 +651,32 @@ def test_affine_transform_inverse_known_values(self):
             numpy.testing.assert_array_almost_equal(src, tform(dst))
 
 
+class AnamorphosisTransformKnownValues(unittest.TestCase):
+
+    @staticmethod
+    def _grid(n):
+        xv = yv = numpy.arange(n, dtype=float) - 0.5 * float(n - 1)
```

Comment:
The `float(...)` is not needed. `0.5 * (n -1)` is a float. BTW, maybe it's clearer to write `(n - 1) / 2`?